flink-end-to-end-tests/README.md

```diff
@@ -5,6 +5,11 @@ This module contains tests that verify end-to-end behaviour of Flink.
 The tests defined in `run-nightly-tests.sh` are run by the CI system on every pull request
 and push to master.
 
+**Before running end-to-end tests, make sure you have built Flink and all modules using:**
```

Comment:
I suggest it would read better if this new text is at the start of the Running Tests section